doc/reference/configuration/cfg_binary_logging_snapshots.rst

```diff
@@ -111,3 +112,26 @@
     | Default: 16777216 bytes
     | Environment variable: TT_WAL_QUEUE_MAX_SIZE
     | Dynamic: **yes**
+
+.. _cfg_binary_logging_snapshots-wal_cleanup_delay:
+
+.. confval:: wal_cleanup_delay
+
+    Since version :doc:`2.8.1 </release/2.8.1>`.
+    The delay (in seconds) used to prevent the :ref:`Tarantool garbage collector <cfg_checkpoint_daemon-garbage-collector>`
+    from immediate removing :ref:`write-ahead log<internals-wal>` files after a node restart.
+    This delay helps :ref:`replicas <replication-roles>` sync with a master faster after its restart and
```

Comment:
I wouldn't say it helps to sync faster: it eliminates the risk that master deletes WALs needed for replicas after a restart. It fixes a possible erroneous situation rather than improves anything.

Although, yes, this means replicas do not risk downloading the data all over again, and thus can sync faster.